### ISSUE TITLE
Determination search by specific epithet only does not work

### ISSUE THREAD

**Issue by michellelocke** [AAFC-BICoE/dina-planning#267]
If I search for only the specific epithet (species level name) in the Scientific Name Search I cannot get the correct response.

Example:

Result I want: Dasysyrphus limatus (Hine, 1922), this is a flower fly of the family Syrphidae.

I searched for "limatus". I only get one option: Limatus (presumably the genus of mosquito, but there is no corroborating information).

In order to get the name I want I have to type out the full name "Dasysyrphus limatus".

The reason this is probelmatic is that a user may only have an initial and a specific epithet (ex: D. limatus) or just the specific epithet (ex: limatus). There may also be spelling mistakes in the genus name on their label; searching by only the specific epithet allows the user to find the correct name despite a mistake in their label. 

The user should be presented with a list of all possible names with limatus in them and also an option to narrow down their search (ex: search within the Family Syrphidae only). 

It will save time not to have to type out the full name (genus + specific epithet) and help to facilitate finding the correct result.

**Comment by dimus** [comment]
I imagine it should be something similar to 
http://gni.globalnames.org/name_strings?search_term=sp%3Alimatus&commit=Search

**Comment by dshorthouse** [comment]
Thanks, @dimus. That's exactly right. Only in this case, we'd be using gnverifier because we're interested in names from sources along with their tree paths, cached for use in local search when available.

**Comment by dimus** [comment]
Yes, @dshorthouse, I would not use GNI, its data is outdated by now. My plan was to introduce 'faceted search' to gnverifier at some point, and if there is a real use-case, it means it will be high on priority list and will take in account DINA's particular needs

**Comment by dimus** [comment]
Newly added faceted search api finds the names with such epithet, but the list, unerstandably is pretty huge:

http://verifier.globalnames.org/api/v0/search/sp:limatus

The list can be trimmed down if more information is known, for example:

http://verifier.globalnames.org/api/v0/search/sp:limatus%20g:D.

or

http://verifier.globalnames.org/api/v0/search/sp:limatus%20tx:Syrphinae

The API is fresh out of the oven, so it is in Beta stage currently, documentation is at

https://apidoc.globalnames.org/gnames-beta

**Comment by dshorthouse** [comment]
Thanks for documenting this @dimus & for the OpenAPI endpoint; this makes it much easier to visualize the scope of issues (& what effect this would have on user experience), especially when there might be inputs that result in an explosion of results, eg `n:P.+minuta`.

As yet another example using the `n:` component:
https://verifier.globalnames.org/api/v0/search/n:D.+limatus+(Hine,+1922)
...produces a single result. Yay!

**Comment by dimus** [comment]
One thing where `n:` approach might be contr-intuitive, is if people try to enter a complex name name as is, and wonder why something they wanted to search for is not used. For example if there is more than one author, only the last one will count. For example here

https://verifier.globalnames.org/api/v0/search/n:B.+limatus+bubo+(Barr, Linn.)

`limatus` is not used because there is subspecies `bubo`, and `Barr` is not used, because there is `Linn.`
